pkg/kubectl/resize.go

```diff
@@ -102,7 +102,7 @@ type ReplicationControllerResizer struct {
 }
 
 type RetryParams struct {
```

Comment:
since this is public, perhaps a godoc line explaining it and what the two members do?